packages/core/src/styling/theme.ts

```diff
@@ -438,3 +444,21 @@ export function renderTheme(
 
   return css
 }
+
+export function getThemeEntries(
+  theme: Theme,
+  themeName: string | undefined,
+): Record<string, string | null> {
+  const entries: Record<string, string | null> = {}
+  if (!themeName) {
+    return entries
+  }
+
+  for (const [propertyName, definition] of Object.entries(
+    theme.propertyDefinitions ?? {},
+  )) {
+    entries[propertyName] = definition.values?.[themeName] ?? null
```

Comment:
Is it useful/relevant to have a lot of `null` entries for a theme? I suppose they might be useful to render in some cases in our UI, but generally I suppose they're not needed?